preferences/src/main/kotlin/ktx/preferences/preferences.kt

```diff
@@ -0,0 +1,43 @@
+package ktx.preferences
+
+import com.badlogic.gdx.Preferences
+import com.badlogic.gdx.utils.GdxRuntimeException
+
+inline operator fun <reified T> Preferences.set(key: String, value: T): Preferences {
```

Comment:
Please revert the previous overloaded setters, this is basically a workaround that we should use for `get` only due to no union types.